http-verbs-common/src/test/scala/uk/gov/hmrc/http/RetriesSpec.scala

```diff
@@ -181,26 +183,28 @@ class RetriesSpec extends AnyWordSpecLike with Matchers with MockitoSugar with S
       }
 
       val mdcData = Map("key1" -> "value1")
-      mdcData.foreach { case (k, v) => MDC.put("key1", "value1") }
-      val mdcEc = new uk.gov.hmrc.play.http.logging.MdcLoggingExecutionContext(implicitly[ExecutionContext], mdcData)
+
+      implicit val mdcEc = ExecutionContext.fromExecutor(new uk.gov.hmrc.play.http.logging.MDCPropagatingExecutorService(Executors.newFixedThreadPool(2)))
```

Comment:
using the bootstrap Mdc implementation here

Reply:
Import uk.gov.hmrc.play.http.logging.MDCPropagatingExecutorService ?